### PR TITLE
feat:[DSSC-8669mcdrw] Integration des Notifications

### PR DESCRIPTION
**Pull request by aertyuiop12345**
Closes #

#### What has been done to verify that this works as intended?
Aucun test unitaires n'a été fait sur les nouvelles insertions de code
#### Why is this the best possible solution? Were any other approaches considered?
non
#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Aucun risque de regression. Les fonctionnalités impactés par ce travail concerne uniquement les boutons et la mise a jour des couleurs
#### Do we need any specific form for testing your changes? If so, please attach one.
aucun
#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
non
#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

**Review by Baom92 — APPROVED** [review]

**Review by jgnacadja — CHANGES_REQUESTED** [review]
reverifier la logique de souscription au service de notification

File: collect_app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -75,6 +60,12 @@ the specific language governing permissions and limitations under the License.
         android:smallScreens="true"
         android:xlargeScreens="true" />
 
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.MAIN" />
+        </intent>
+    </queries>
```
Comment (Baom92) [inline collect_app/src/main/AndroidManifest.xml]: Cette modification sert à quoi ?

File: collect_app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -84,8 +75,12 @@ the specific language governing permissions and limitations under the License.
         android:largeHeap="true"
         android:networkSecurityConfig="@xml/network_security_config"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="true"
-        android:theme="@style/Theme.Collect">
+        android:theme="@style/Theme.Collect"
+        android:usesCleartextTraffic="true">
+        <receiver
```
Comment (Baom92) [inline collect_app/src/main/AndroidManifest.xml]: Celà sert à quoi ?

File: collect_app/src/main/AndroidManifest.xml
Comment (Baom92) [inline collect_app/src/main/AndroidManifest.xml:1]: Pourquoi toutes ces modifications dans ce fichier ?

File: collect_app/src/main/java/org/odk/collect/android/UninstallReceiver.java
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/UninstallReceiver.java:1]: Il sert à quoi ?

File: collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt [hunk 3]
```diff
@@ -103,4 +109,37 @@ class FirstLaunchActivity : CollectAbstractActivity() {
             }
         }
     }
-}
\ No newline at end of file
+
+    private fun subscribeToWP() {
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt:113]: La souscription sera effectué chaque fois. C'est pas logique

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 4]
```diff
@@ -56,7 +58,10 @@ public void onClick(View v) {
         messageButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+<<<<<<< HEAD
                 confirmOpenForum();
+=======
+>>>>>>> e4028c08f (feat: [DSSC#8669mcdrw])
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: ????

**Review by Baom92 — CHANGES_REQUESTED** [review]

**Comment by Baom92** [comment]
Réponds à mes questions stp.
Et ne résout pas mes conversations

**Review by Baom92 — APPROVED** [review]

**Pull request merged**